docs/reference/ml/apis/dfanalyticsresources.asciidoc

```diff
@@ -0,0 +1,49 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-dfanalytics-resources]]
+=== {dfanalytics-cap} resources
+
+A {dfanalytics} configuration object has the following properties:
+
+`analysis`::
+  (object) The type of analysis that is performed on the `source`. For example: 
+  `outlier_detection`.
+
+`dest`::
```

Comment:
Instead of explaining the fields here, I think we should add a link to a section that describes the object and list the properties like we do for jobs and datafeeds. The same applies to `source`.